css/src/tokens/palette.scss

```diff
@@ -0,0 +1,184 @@
+$palette-green-70: hsla(125, 80%, 22%, 1) !default; // #0b6413
+$palette-green-80: hsla(135, 88%, 16%, 1) !default; // #054b16
+$palette-green-90: hsla(129, 83%, 11%, 1) !default; // #05350c
+$palette-green-100: hsla(106, 100%, 5%, 1) !default; // #061a00
+
+$palette-green-opacity-30: hsla(120, 37%, 45%, 0.3) !default;
+$palette-green-opacity-70: hsla(120, 37%, 45%, 0.7) !default;
+
+// Purple
+
+$palette-purple-10: hsla(277, 90%, 92%, 1) !default; // #efd9fd
+$palette-purple-20: hsla(275, 97%, 86%, 1) !default; // #e0b7fe
+$palette-purple-30: hsla(274, 100%, 81%, 1) !default; // #d59dff
+$palette-purple-40: hsla(268, 62%, 69%, 1) !default; // #ac7ee1
+$palette-purple-50: hsla(265, 53%, 63%, 1) !default; // #9970d3
+$palette-purple-60: hsla(262, 46%, 58%, 1) !default; // #8661c5
+$palette-purple-70: hsla(261, 33%, 43%, 1) !default; // #624991
+$palette-purple-80: hsla(259, 31%, 26%, 1) !default; // #3b2e58
+$palette-purple-90: hsla(251, 47%, 18%, 1) !default; // #201843
+$palette-purple-100: hsla(244, 100%, 9%, 1) !default; // #03002c
+
+$palette-purple-opacity-30: hsla(251, 47%, 18%, 0.3) !default;
+$palette-purple-opacity-70: hsla(251, 47%, 18%, 0.7) !default;
+
+// Yellow
+
+$palette-yellow-10: hsla(47, 100%, 90%, 1) !default; // #fff4ce
+$palette-yellow-20: hsla(45, 100%, 81%, 1) !default; // #ffe79f
+$palette-yellow-30: hsla(44, 100%, 76%, 1) !default; // #ffdf84
+$palette-yellow-40: hsla(44, 100%, 62%, 1) !default; // #ffcb3f
+$palette-yellow-50: hsla(44, 100%, 50%, 1) !default; // #ffb900
+$palette-yellow-60: hsla(43, 99%, 41%, 1) !default; // #d19501
+$palette-yellow-70: hsla(41, 97%, 30%, 1) !default; // #966802
+$palette-yellow-80: hsla(38, 66%, 25%, 1) !default; // #6a4b16
+$palette-yellow-90: hsla(35, 70%, 18%, 1) !default; // #4f340e
+$palette-yellow-100: hsla(29, 88%, 9%, 1) !default; // #2d1703
+
+$palette-yellow-opacity-30: hsla(44, 100%, 50%, 0.3) !default;
+$palette-yellow-opacity-70: hsla(44, 100%, 50%, 0.7) !default;
+
+// Red
+
+$palette-red-10: hsla(355, 85%, 95%, 1) !default; // #FDE7E9
+$palette-red-20: hsla(359, 54%, 78%, 1) !default; // #E5A7A8
+$palette-red-30: hsla(359, 51%, 65%, 1) !default; // #D4797A
+$palette-red-40: hsla(0, 50%, 54%, 1) !default; // #C54F4F
+$palette-red-50: hsla(0, 100%, 33%, 1) !default; // #B62626
+$palette-red-60: hsla(0, 100%, 33%, 1) !default; // #A80000
+$palette-red-70: hsla(0, 100%, 26%, 1) !default; // #870000
+$palette-red-80: hsla(359, 100%, 19%, 1) !default; // #630001
+$palette-red-90: hsla(359, 100%, 14%, 1) !default; // #470001
+$palette-red-100: hsla(359, 100%, 8%, 1) !default; // #290001
+
+$palette-red-opacity-30: hsla(0, 100%, 33%, 0.3) !default;
+$palette-red-opacity-70: hsla(0, 100%, 33%, 0.7) !default;
+
+// High Contrast
+
+$palette-yellow-high-contrast: hsla(60, 100%, 50%, 1) !default;
+$palette-yellow-high-contrast-hover: hsla(60, 100%, 60%, 1) !default;
+$palette-visited-high-contrast: hsla(106, 100%, 50%, 1) !default;
```

Comment:
@wibjorn Curious why several of these are defined as `hsla()` with the HEX as a comment? Why not define as the HEX?